velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -195,43 +217,118 @@ class MinMaxByAggregationTestBase : public AggregationTestBase {
 // Build a flat vector with StringView. The value in the returned flat vector
 // is in ascending order.
 template <>
-FlatVectorPtr<StringView> MinMaxByAggregationTestBase::buildDataVector() {
+FlatVectorPtr<StringView> MinMaxByAggregationTestBase::buildDataVector(
+    vector_size_t size,
+    folly::Range<const int*> values) {
+  std::string value;
+  if (values.empty()) {
+    return makeFlatVector<StringView>(
```

Comment:
If the point is to build a random string vector that can also be done by fuzzer, infact it even has unicode support; which you must test against. Please use that.

Reply:
No. This is to a ordered vector here.